### PR TITLE
Student list component

### PR DESCRIPTION
**Pull request by AngelaLuo49021**
Screen Shot 2020-03-07 at 9 34 04 PM

## Description :star2:

Fixes #<number>

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -3,11 +3,15 @@ import "./App.css";
...
 import { compose } from "recompose";
 import ExampleComponent from "ExampleComponent/ExampleComponent";
+import AssignmentPage from "../AssignmentPage/AssignmentPage";
+import StudentList from "../StudentList/StudentList";
 
 function App() {
   return (
     <div className="App">
-      <ExampleComponent />
+      {/* <ExampleComponent /> */}
```
Comment (chyku) [inline src/components/App/App.js]: can u delete this

File: src/components/App/App.js [hunk 2]
```diff
@@ -3,11 +3,15 @@ import "./App.css";
...
+      <AssignmentPage />
+      {/* <StudentList/> */}
```
Comment (chyku) [inline src/components/App/App.js]: also delete this

File: src/components/StudentList/StudentListStyles.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  listSection: {
+    backgroundColor: "#e7e4e4",
+    color: "#00d8a4"
+  },
+  nested: {
+    backgroundColor: "#7f7eff",
+    color: "#e6e6e6",
+    paddingLeft: theme.spacing(4),
+    "&:hover": {
+      backgroundColor: "#00d8a4"
```
Comment (chyku) [inline src/components/StudentList/StudentListStyles.js:22]: why is it like.. in a  js instead of css file

**Review by chyku — CHANGES_REQUESTED** [review]

**Review by gwang0136 — APPROVED** [review]
⭐️

File: src/pages/AssignmentPage/AssignmentPage.js [hunk 4]
```diff
@@ -1,35 +1,48 @@
...
-            <Col>
-              <DatePicker handleChange={handleDatePickerChange} />
-            </Col>
-          </Row>
-        </Container>
-      </div>
+    <div className="place_middle">
+      <Container>
+        <Row>
+          <h1>this is the assignment page yall</h1>
```
Comment (chyku) [inline src/pages/AssignmentPage/AssignmentPage.js:32]: // todo: change this

**Review by chyku — APPROVED** [review]

**Pull request merged**